py/server/deephaven/table.py

```diff
@@ -387,7 +387,12 @@ def _encode_signature(fn: Callable) -> str:
 
     If a parameter or the return of the function is not annotated, the default 'O' - object type, will be used.
     """
-    sig = inspect.signature(fn)
+    try:
+        sig = inspect.signature(fn)
+    except:
+        if type(fn) == np.ufunc:
+            return "O"*fn.nin + "->" + "O"
+        return "->O"
```

Comment:
Are the output types just PyObject?  If so, that's not ideal.  Can we do better?

Reply:
Yes, but that's left for #4762.